frontend-css-concept/styles/styles.css

```
@@ -0,0 +1,414 @@
+}
+
+img {
+    width: 100%;
+}
+
+:root {
+    --chrysler-blue: #5722cdff;
+    --majorelle-blue: #4b49e8ff;
+    --magnolia: #f2f0ffff;
+    --french-rose: #ed4385ff;
+    --night: #0c0c0cff;
+    --licht-grey: #F3F6F9;
+    --grey:  #7A7A7A;
+}
+
+
+body {
+    font-family: 'Roboto', 'Arial', sans-serif;
+    color: var(--night);
+}
+
+input,
+button,
+form {
+    font-family: inherit;
+    font-size: inherit;
+}
+
+h1,
+h2,
+h3 {
+    font-family: 'Merriweather', 'Times New Roman', serif;
+}
+
+h1 {
+    font-size: 3rem;
+}
+
+a,
+a:visited {
+    text-decoration: none;
+    color: var(--chrysler-blue);
+}
+
+a:hover {
+    cursor: pointer;
+}
+
+
+
+button {
+    background-color: var(--french-rose);
+    color: var(--magnolia);
+    text-transform: uppercase;
+    padding: 1rem 1rem;
+    border-radius: 30px;
+    box-shadow: 2px 2px 3px 2px rgba(0, 0, 0, 0.2);
+    border: 1px solid var(--french-rose);
+    cursor: pointer;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+header {
+    background: rgb(72,69,228);
+    background: linear-gradient(90deg, rgba(72,69,228,1) 52%, rgba(87,34,205,1) 100%);
+    color: var(--magnolia);
+    font-weight: 300;
+    position: relative;
+}
+
+.header-column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+}
+
+header .inner-container {
+    flex-basis: 320px;
+    max-width: 600px;
+    padding: 0 0 6rem 0;
+    margin: 4rem 0;
+}
+
+header h1 {
+    text-align: center;
+    padding: .5em;
+}
+
+
+.navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    flex-basis: auto;
+    flex-shrink: 1;
+    flex-grow: 0;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+.navigation ul li {
+    display: inline-block;
+    padding: 1em 2em;
+}
+
+.navigation a {
+    color: var(--magnolia);
+}
+
+
+.header-squiggle {
+    position: absolute;
+    width: 3.5rem;
+    bottom: 0;
+}
+
+header h1 {
+    margin: 0.5rem;
+}
+
+header p {
+    font-size: 1.2rem;
+}
+
+header button {
+    width: 12rem;
+    margin: 1.5rem;
+    font-size: 0.7em;
+    letter-spacing: 0.1em;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.hello {
+    color: var(--chrysler-blue);
+}
+
+.hello .inner-container {
+    width: 600px;
+    margin: 4rem 4rem;
```

Dit kun je nog korter schrijven: 
```suggestion
    margin: 4rem;
```